tests/test_nodetree.py

```diff
@@ -399,6 +399,45 @@ def test_get_wildcard(connection):
     result = tree.demods(deep=True)
     assert result[tree.demods[0].impedance] == (None, 125)
 
+    connection.get.return_value = OrderedDict(
+        [
+            (
+                "/dev1234/demods/0/impedance",
+                {
+                    "timestamp": array("q", [3880906635]),
+                    "x": array("l", [123]),
+                    "y": array("l", [123]),
+                },
+            )
+        ]
+    )
+    result = tree.demods()
+    assert result[tree.demods[0].impedance] == connection.get.return_value["/dev1234/demods/0/impedance"]
+    connection.get.assert_called_with("/dev1234/demods", settingsonly=False, flat=True)
+
+def test_module_get_wildcard(connection):
```

Comment:
nitpick: This test could be parametrized so only 1 assert would exists.

Otherwise LGTM!